Src/Support/Google.Apis.Auth/OAuth2/ServiceAccountCredential.cs

```diff
@@ -194,6 +194,13 @@ public ServiceAccountCredential(Initializer initializer) : base(initializer)
         /// The <paramref name="credentialData"/> does not contain valid JSON service account key data.
         /// </exception>
         /// <returns>The credentials parsed from the service account key data.</returns>
+        /// <remarks>
+        /// Important: If you accept a credential configuration (credential JSON/File/Stream) from an external source
+        /// for authentication to Google Cloud, you must validate it before providing it to any Google API or library.
+        /// Providing an unvalidated credential configuration to Google APIs can compromise the security of your
+        /// systems and data. For more information, refer to
+        /// <see href="https://cloud.google.com/docs/authentication/external/externally-sourced-credentials">Validate credential configurations from external sources</see>.
+        /// </remarks>
```

Comment:
Do we need to update here? Only SA json can be accepted here right?

Reply:
Started an internal thread.

Reply:
We discussed internally, removing from here as it's not necessary.